soroban-env-host/src/budget.rs

```diff
@@ -233,19 +233,19 @@ impl Default for BudgetImpl {
                 // whatever function call, arg-shuffling, spills, reloads or
                 // other flotsam accumulates around a typical memory copy.
                 ContractCostType::MemCpy => {
-                    cpu.const_term = 250;
+                    cpu.const_term = 50;
                     cpu.lin_term = ScaledU64((1 << COST_MODEL_LIN_TERM_SCALE_BITS) / 8);
                 }
                 ContractCostType::MemCmp => {
-                    cpu.const_term = 250;
+                    cpu.const_term = 50;
                     cpu.lin_term = ScaledU64((1 << COST_MODEL_LIN_TERM_SCALE_BITS) / 8);
                 }
                 ContractCostType::DispatchHostFunction => {
-                    cpu.const_term = 263;
+                    cpu.const_term = 1368;
```

Comment:
I believe this is a mistake -- or rather a result of having `feature = "testutils"` enabled during testing, which is creating an (expensive) input to the host lifecycle hook. We need to un-measure this because it's a significant overcharge from what dispatching a host function will cost in production without `feature = "testutils"` (and if you notice below it also charges _memory_ which should not happen)

Reply:
Thanks for the good catch! I have split all calibration related code to a new feature "bench" such that it's decoupled from "testutils". Now the numbers for `DispatchHostFunction` and `I256` arithmetics have gone down to previous levels.